src/JuliusSweetland.OptiKey/UI/ViewModels/MainViewModel.ServiceEventHandlers.cs

```diff
@@ -619,35 +619,37 @@ private void HandleFunctionKeySelectionResult(KeyValue singleKeyValue)
                     break;
 
                 case FunctionKeys.MouseKeyboard:
-                    Log.Info("Changing keyboard to Mouse.");
-                    Action backAction;
-                    if (keyStateService.SimulateKeyStrokes
-                        && Settings.Default.SuppressModifierKeysWhenInMouseKeyboard)
                     {
-                        var restoreModifierStates = keyStateService.ReleaseModifiers(Log);
-                        backAction = () =>
+                        Log.Info("Changing keyboard to Mouse.");
```

Comment:
backAction. Lots of cases need to create a backAction, and I was getting tired of coming up with new suffices for e.g. backActionMouse, backActionMinecraftKeyboard. I'll update the commit log.